schema/spacewalk/upgrade/susemanager-schema-4.2.4-to-susemanager-schema-4.2.5/900-centos6-vault.sql

```diff
@@ -0,0 +1,4 @@
+--- After CentOS EoL, it got removed from the mirrorlist and moved to the vault
+UPDATE rhnContentSource
+SET source_url = concat('https://vault.centos.org/centos/', array_to_string(regexp_matches(source_url, 'repo=(.+)', 'g'), ''), '/' ,array_to_string(regexp_matches(source_url, 'arch=(.+)&', 'g'), ''), '/')
+WHERE source_url LIKE 'http://mirrorlist.centos.org/?release=6%';
```

Comment:
Hmm, can you limit it to channels which are not vendor channels?
AFAIK rhnContentSource has an `org_id` column. That column should not be NULL.

I tried that a `mgr-sync refresh` updates the URLs of vendor channels correctly.
So a schema migration should not be needed for this case.

Reply:
Checking...

Reply:
Yes, you are right. According to what I see at the table, only repos for custom channels have a value.